### ISSUE TITLE
add comments to words

### ISSUE THREAD

**Issue by arrocke** [arrocke/gloss-translation#305]
* In the sidebar, a user can click the add comment button to get a rich text editor to add a comment
* The user, timestamp, and comment (as HTML) are saved to the database
* The list of comments is shown in the sidebar
* We will use tiptap for our rich text editor. We need to support bold, italic and lists

![Image](https://github.com/arrocke/gloss-translation/assets/11968925/287c03cb-b128-48ba-9344-c64bbcb859e3)

**Comment by Pertempto** [comment]
Can we split this into a few smaller issues? It seems pretty large in scope.

**Comment by arrocke** [comment]
This is just showing the list of comments and adding new threads. Replying and resolving are in other issues. Do you want to break this up even more?

**Comment by Pertempto** [comment]
> Replying and resolving are in other issues.

Ok good.

**Comment by arrocke** [comment]
@Pertempto We need to hold on this for a bit. There are other epics that need to be completed before we can onboard translation teams, and this isn't one of them
